docs/database_schema_diagrams/source/work_entity_details.json

```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables for the “work” core entity properties",
-  "_comment": "It shows replicated tables used to define any work entity, beyond relationships. It can be used as an example for tables generally used to define any entity of any type. The core entity is highlighted.",
+  "tooltip": "Tables for the “work” entity properties",
```

Comment:
Others say "entity type"

Reply:
Good catch, now fixed in commit https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/2e2bba792d614a75c0e15a36c80aacca6edd53af.